diesel_derives/src/field.rs

```diff
@@ -65,6 +65,23 @@ impl Field {
             })
     }
 
+    pub fn column_str_name(&self) -> String {
+        self.column_name_from_attribute
+            .as_ref()
+            .map(|m| {
+                m.str_value().unwrap_or_else(|e| {
+                    e.emit();
+                    "unknown_column".to_string()
```

Comment:
If its a named field we should use the actual name of the field here, similary that done in the existing `column_name` function.